dashboard/app/models/levels/level.rb

```diff
@@ -615,7 +625,18 @@ def summarize_for_edit
       type: self.class.to_s,
       name: name,
       updated_at: updated_at.localtime.strftime("%D at %r"),
-      owner: user&.name
```

Comment:
`summarize_for_edit` is used to summarize the levels which are provided when you search for levels. This updates the information provided to cover the information we need in order to edit the level. @davidsbailey It would be great to double check that this matches the stuff you having been working on.

Reply:
This looks like the right approach. Since there are multiple contexts a level can be edited in (script edit, lesson edit, and editing the level itself) I think it could be helpful to qualify in the method name here. so I would suggest renaming this to `summarize_for_lesson_edit` or possibly `summarize_for_add` or something like that.  In order to move away from sending down all lesson keys, I am going to add a new method soon called `summarize_for_script_edit`, for example, because i will want to send down the level key there but probably not in other places.